css/style.css

```diff
@@ -1,6 +1,180 @@
+* {
+  box-sizing: border-box;
+}
+
 html,
 body {
-  font-family: montserrat-font, sans-serif;
+  font-family: 'montserrat-font', sans-serif;
   font-size: 15px;
   font-weight: normal;
+  color: #000000;
```

Comment:
Codacy found an issue: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/GurevAleksey_itogovaya/pullRequest?prid=6093420)